datasette/static/extra_styles.css

```
@@ -0,0 +1,30 @@
+pre, code {
+    font-family:
+    "SFMono-Regular",
+    Consolas,
+    "Liberation Mono",
+    Menlo,
+    Courier,
+    monospace;
+
+}
+
+pre.code-sample {
+    background-color: white;
+}
+
+pre.code-sample>code {
+    display: inline-flex;
+}
+
+pre.code-sample>code:before {
+    counter-increment: line;
+    content: counter(line);
+    border-right: 1px solid #aaaaaa;
+    padding-right: .5em;
+    margin-right: .5em;
+    padding-left: .25em;
+    color: #999;
+    width: fit-fontent(3ch);
```

I will fix this though 🤦 
```suggestion
    width: fit-content(3ch);
```